src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -75,12 +75,12 @@ public void Initialize(TelemetryConfiguration configuration)
                 if (this.options.Tracing.Version == Options.DurableDistributedTracingVersion.V2)
                 {
                     DurableTelemetryModule module = new DurableTelemetryModule();
-                    module.Initialize(telemetryConfiguration);
-                    this.telemetryModule = module;
+                    module.Initialize(configuration);
+                    this.TelemetryModule = module;
 
                     WebJobsTelemetryModule webJobsModule = new WebJobsTelemetryModule();
```

Comment:
Small design suggestion: we could reduce down to needing only 1 telemetry module with a minor change: `WebJobsTelemetryModule` can wrap/include the `DurableTelemetryModule` internally, and forward the `Initialize` call to it.

Reply:
Thanks for the suggestion! I'll make this change in a follow up PR. I tracked this change in GitHub issue https://github.com/Azure/azure-functions-durable-extension/issues/3038.

Reply:
You don't have to make the change at all if you don't want.